pkg/engine/files.go

```diff
@@ -156,5 +156,5 @@ func (f files) Lines(path string) []string {
 		return []string{}
 	}
 
-	return strings.Split(string(f[path]), "\n")
+	return strings.Split(strings.TrimRight(string(f[path]), "\n"), "\n")
```

Comment:
Apologies for nitpicking, but this will remove all empty lines at the end. e.g.

```
foo\n
bar\n
\n
\n
\n
```

Would now result in `["foo", "bar"]`. Previously, this would have been `["foo", "bar", "", "", "", ""]`.

OTOH, I agree with the setiment. A user with a file:
```
foo\n
bar\n
```

Almost certainly wants `["foo", "bar"]`.

A more correct, and perhaps simple fix. Where the user is in control, might be:
```
{{- range .Files.Lines "paths.list" | without "" }}
- path: {{ . }}
  pathType: ImplementationSpecific
{{- end }}
```

I'm not necessary against this PR. I do think we should consider not stripping multiple new lines. But also as above, letting the user choose also might be best.

Reply:
> I think even in your example, the expectation would be to have only three blank entries, not four. There are only three blank lines.

heh, yes. Off by one error from me.

Reply:
I have made this change.